### PR TITLE
Feature/recon

### PR DESCRIPTION
**Pull request by jkesanie**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -185,9 +185,77 @@ def calculate_date_range(start, end, intv):
...
+            },
+            {
+                "id": "Place",
+                "name": "Place"
+            },
+        ]
+    }
+
+@app.post('/recon/reconcile',
+    response_model=ReconResponse,
```
Comment (sennierer) [inline main.py:228]: there seems to be something wrong with the response model. at least in my local instance the swagger definitone does not show a proper response shape

File: query_parameters.py [hunk 2]
```diff
@@ -94,6 +94,36 @@ def __post_init__(self):
                 self.diedAfter).strftime('%Y-%m-%dT00:00:00'))
 
 
+
+
+@dataclasses.dataclass()
+class ReconQuery(Base):
+    query: str   
+    limit: str
+    type: str
```
[the remark this answers is not in the document]
Comment (sennierer) [reply query_parameters.py:103]: also if adding an enum of allowed values there wouldnt be any need to do manual validation in the view

**Review by sennierer — CHANGES_REQUESTED** [review]
could you try to add the the proper model definitions (left some comments in the code), so that we get a valid swagger definition in the API?
Also: is that manifests endpoint needed if we have an proper swagger endpoint delivering a definition of the whole API?
and as a side note: I didnt find any "Franz" (very popular German first name) in the API. Looking at the sparql query it seems to be fine, so probably a problem in the data, do you have an idea where the problem could be?

**Comment by jkesanie** [comment]
The manifest is part of the reconciliation spec so I think it needs to be there.

**Comment by sennierer** [comment]
ok, then lets leave the manifest endpoint there

File: query_parameters.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jkesanie) [reply query_parameters.py:103]: The problem is that I don't know how to add stuff to the model in a way that is not visible in the Swagger spec but usable in the template. I want the type parameter to have simple values (Person) which are turned into URI and used in the template. How should this be done? That's the view ifs are currently using URI and and enum for type can't use the original simple values.

**Pull request merged**